client/pages/AssessmentSection/Descriptions/CommentableDescription/CommentableDescription.tsx

```diff
@@ -40,9 +41,7 @@ const CommentableDescription: React.FC<Props> = (props) => {
         />
       </div>
       <div className="fra-description__review-indicator-wrapper">
-        {!disabled && (
-          <pre>{`<ReviewIndicator section={section} title={title} target={[name]} countryIso={countryIso} />`}</pre>
-        )}
+        {!disabled && <ReviewIndicator title={title} topicKey={name} />}
```

Comment:
What should we use here as topicKey?
countryIso + section + name?

Should we add a helper (eg. `Topics.getKey...()`)?

Reply:
CountyIso + assessmentName + cycleName + sectionName + name 

And yes add 
getKeyComments

Reply:
Under descriptions or topics ?